recipe/conda_build_config.yaml

```
@@ -1,19 +1,21 @@
 channel_targets:
   - conda-forge rubin-env_dev
 c_compiler_version:            # [unix]
-  - 12                         # [linux]
-  - 16                         # [osx]
+  - 13                         # [linux]
+  - 18                         # [osx]
 cxx_compiler_version:          # [unix]
-  - 12                         # [linux]
-  - 16                         # [osx]
+  - 13                         # [linux]
+  - 18                         # [osx]
 fortran_compiler_version:      # [unix]
-  - 12                         # [linux]
-  - 12                         # [osx]
+  - 13                         # [linux]
+  - 13                         # [osx]
 llvm_openmp:                              # [osx]
-  - 16                                    # [osx]
+  - 18                                    # [osx]
 python:
-  # part of a zip_keys: python, python_impl, numpy
-  - 3.11.* *_cpython
+  # part of a zip_keys: python, python_impl, numpy, is_python_min
+  - 3.12.* *_cpython
+is_python_min:
+  - true
```

```suggestion
  - false
```

This should be false.

---

I can't find from quick search what `is_python_min` means? Is this saying that if true then 3.12 and 3.13 will work but if false it's pinned to 3.12? Is there any harm in defining a rubin-env that also works on 3.13? (assuming the packages have been migrated).

---

it is used to mark which python version is the minimum supported one on conda-forge. that version is 3.9. So it should be False.